doc/nrf/app_dev/device_guides/nrf54l/kmu_provision.rst

```diff
@@ -0,0 +1,42 @@
+.. _ug_nrf54l_developing_provision_kmu:
+
+nRF54L15 KMU provisioning
+#########################
+
+nRF54L15 is equipped with Hardware Key Management Unit which needs to be provisioned if is in Usage.
+NCS provide west command ```ncs-provision`` which allows to upload keys to the device thought SWD interface.
+
+Prerequisite
+************
+
+nrfprovision script need to be installed
+
+.. parsed-literal::
+   :class: highlight
+
+    pip install nrfprovision
+
+
+Provisioning keys to the board
+******************************
+
+.. parsed-literal::
+   :class: highlight
+
+    west ncs-provision upload -s nrf54l15 -k ed25519.pem -k ed25519-1.pem -k ed25519-2.pem
+
+Parameter -s (-–soc) specifies target device, currently nrf54l15 is only supported.
+
+Parameters -k (-–key) specifies private key pem files of keys which have to be provisioned to the SoC. This parameters might be specified up to tree times.
+The script generates the public key for each private key and upload these public keys to the device.
+These public keys populating generations of the Key for verification of the application image verified by the MCUboot.
+The first key in the command is relevant for the first generation - is shall be used for the application image signing.
+Currently the script supports only ED25519 Keys.
+
+The command shall be used on not provisioned SoC.
+If the SoC was provisioned before, and different keys set shall be use the SoC should be erased with recovery command.
+
+.. parsed-literal::
+   :class: highlight
+
+    west flash --recover
```

Comment:
not working without --build-dir option
(and user does not need to run --recover, --erase option during flashing is enough to clear the keys)

instead of using west flash, user can run
`nrfjprog --eraseall`
or
`nrfutil device erase` (plus --serial-number if more boards is connected)